types/mapbox__mapbox-sdk/index.d.ts

```diff
@@ -1312,7 +1312,10 @@ declare module '@mapbox/mapbox-sdk/services/static' {
         padding?: string | undefined;
         overlays?: Array<CustomMarkerOverlay | SimpleMarkerOverlay | PathOverlay | GeoJsonOverlay> | undefined;
         highRes?: boolean | undefined;
-        insertOverlayBeforeLayer?: string | undefined;
+        before_layer?: string | undefined;
+        addlayer?: object | undefined;
+        setfilter?: any[] | undefined;
```

Comment:
Are there more specific types available for these properties?

Reply:
Good point. The `mapboxgl` package provides an `AnyLayer` type, so in 5d495ca I've swapped it as the type for `addlayer`. There's no more specific type for `setfilter`, however. Refer to this: 

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/16ef4c4bc4829e8c7dc9bd4fdcdd5e2ab353ac4d/types/mapbox-gl/index.d.ts#L355